src/app/components/SkipToContentWrapper.spec.tsx

```diff
@@ -18,17 +18,41 @@ function renderToDom<P>(component: ReactElement<P>) {
     return {component: c, node};
 }
 
+// JSDom logs to console.error when an Error is thrown.
+// Disable the console just in this instance, and re-enable after.
+// See https://github.com/facebook/jest/pull/5267#issuecomment-356605468
+function expectError(message: string, fn: () => void) {
```

Comment:
can we move this to `src/test/reactutils.ts` or maybe invent a `src/test/utils.ts`?